# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../webkit/dumprendertree.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Index: WebKitTools/DumpRenderTree/DumpRenderTree.qtproj/DumpRenderTree.pro
===================================================================
--- ./WebKitTools/DumpRenderTree/DumpRenderTree.qtproj/DumpRenderTree.pro (Revision 21424)
+++ ./WebKitTools/DumpRenderTree/DumpRenderTree.qtproj/DumpRenderTree.pro (Arbeitskopie)
@@ -1,9 +1,9 @@
 TARGET = DumpRenderTree
 
 include(../../../WebKit.pri)
-INCLUDEPATH += /usr/include/freetype2
-INCLUDEPATH += ../../../JavaScriptCore/kjs
 
+CONFIG += link_pkgconfig
+PKGCONFIG += freetype2 fontconfig
 
 QT = core gui
 
